packages/common-ui-web/src/components/Login/Login.styles.ts

```diff
@@ -7,7 +7,11 @@ export const styles: Styles = {
     display: 'flex',
     flexDirection: 'column',
     justifyContent: 'center',
+    gap: '1.5em',
```

Comment:
Use pixels instead of `em`, you're right in the fact that `em` are better for responsiveness, but unfortunately, it can lead to unexpected behavior on certain apps for our clients